0-email-signup-form/FCCvote-net-neutrality-thanks/BSD-but-html-formatted.html

```diff
@@ -0,0 +1,768 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
+<!-- =================== WRAPPER (Header) STARTS =================== -->
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <title>sendto.mozilla.org  |  2015 FCCvote Net Neutrality Thanks</title>
+    <link rel="canonical" href="https://sendto.mozilla.org/page/s/FCCvote-net-neutrality-thanks">
+    <link rel="stylesheet" href="//s.bsd.net/page/static/20150128.2/skins/default/signup/frontend.css" type="text/css">
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.4.1/jquery.min.js" type="text/javascript"></script><script type="text/javascript">!window.jQuery && document.write(unescape('%3Cscript src="/ext/jquery/jquery-1.4.1.min.js"%3E%3C/script%3E'))</script><script src="//s.bsd.net/page/static/20150128.2/ext/jquery/cookie/jquery.cookie.min.js" type="text/javascript"></script><script src="/modules/spud/js/Spud.js.php" type="text/javascript"></script><script type="text/javascript">window.bQuery = jQuery.noConflict(true);</script>
```

Comment:
oh wait am i supposed to be reviewing this specific file? or is it just a compiled one?